savoirfairelinux/sous-chef
# Fix #613: In client list view filter, support multiple statuses.

## Fixes #613 by lingxiaoyang

### Changes proposed in this pull request:

* Change filter type from Choice to MultipleChoice
* Adjust the position of this field.

Preview:
![image](https://cloud.githubusercontent.com/assets/8630726/23321346/1d2ba338-faad-11e6-907c-d85c25157702.png)


### Status


### How to verify this change

1. Go to http://localhost:8000/member/list/?name=&route=&delivery_type=O&display=list&status=D&status=A&status=S&status=C

2. You should be able to see multiple statuses selected there. You can play with it to check if it works well.

### Deployment notes and migration

none

### New translatable strings

none

### Additional notes
none
